ios/FluentUI/Notification/MSFNotificationView.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// UIKit wrapper that exposes the SwiftUI `Notification` implementation
+@objc open class MSFNotification: NSObject, FluentUIWindowProvider {
+
+    /// The UIView representing the Notification.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    /// Creates a new MSFNotification instance.
+    /// - Parameters:
+    ///   - style: The MSFNotification value used by the Notification.
+    ///   - message: The primary text to display in the Notification.
+    ///   - theme: The FluentUIStyle instance representing the theme to be overriden for this Notification.
+    @objc public init(style: MSFNotificationStyle,
+                      message: String,
+                      delayTime: TimeInterval,
+                      theme: FluentUIStyle? = nil) {
+        super.init()
+
+        notification = NotificationViewSwiftUI(style: style, message: message, delayTime: delayTime)
+        hostingController = FluentUIHostingController(rootView: AnyView(notification
+                                                                            .windowProvider(self)
+                                                                            .modifyIf(theme != nil, { notification in
+                                                                                notification.customTheme(theme!)
+                                                                            })))
+        hostingController.view.backgroundColor = UIColor.clear
+        hostingController.disableSafeAreaInsets()
+    }
+
+    public var state: MSFNotificationState {
```

Comment:
An earlier commit was giving me errors (Property cannot be marked @objc because its type cannot be represented in Objective-C), but the lines causing that error have been changed and I missed putting it back!